Guidelet/GuideletLib/UltraSound.py

```diff
@@ -4,12 +4,13 @@
 import time
 
 class UltraSound(object):
-
   DEFAULT_IMAGE_SIZE = [800, 600, 1]
 
+  # -----------------------------------------------------
```

Comment:
In PyCharm: File / Settings / Editor / General / Appearance / Show method separators.
You can configure it to have a stronger color: File / Settings / Editor / General / Editor / Colors & Fonts / General / Code / Method separator color.